beanstalk_worker/worker_start_test.py

```diff
@@ -0,0 +1,136 @@
+#!/usr/bin/env python
+
+import beanstalkc
+from binascii import hexlify
+import hashlib
+import json
+from subprocess import Popen
+from subprocess import PIPE
+import re
+import time
+import logging
+import sys
+import os
+import config
+
+
+bs = beanstalkc.Connection(host="BEANSTALK_SERVER")
+bs.watch("start_test")
+bs.use("test_failed")
+
+config.load_logger()
+logger = logging.getLogger("delivery-worker")
+
+config_path = os.path.dirname(os.path.realpath(__file__))
+kubeconfig = " --config=" + os.path.join(config_path, "node.kubeconfig")
+
+
+def notify_build_failure(name_space, notify_email, logs):
+    msg_details = {}
+    msg_details['action'] = 'notify_user'
+    msg_details['subject'] = "FAILED: Container-build failed " + name_space
+    msg_details['msg'] = "Container build " + name_space + \
+        " failed due to error in build or test steps. Pleae check attached logs"
+    msg_details['logs'] = logs
+    msg_details['notify_email'] = notify_email
+    bs.use('master_tube')
+    bs.put(json.dumps(msg_details))
```

Comment:
-1

The email contents and processing should happen in worker_notify_user module.